### ISSUE TITLE
Error in using ATAppUpdater into project

### ISSUE THREAD

**Issue by jp-dubey07** [apptality/ATAppUpdater#5]
Approach 1:
I have added ATAppUpdater folder in project (screenshot 1). Now, when I try to #import "ATAppUpdater.h" file in view controller it is not available. It forces to use #import "ATAppUpdater/ATAppUpdater.h" (screenshot 2). Now, when application is build it prompts several errors (screenshot 3)

Approach 2:
Since Reachability file is already present in application I have imported only ATAppUpdater .h/.m file.
After building application it prompts error (screenshot 4)

Any fix?

![screen shot 2015-06-03 at 12 15 17 pm](https://cloud.githubusercontent.com/assets/5441735/7954397/e50a691a-09ec-11e5-81a3-4e4c178b06d1.png)
![screen shot 2015-06-03 at 12 15 42 pm](https://cloud.githubusercontent.com/assets/5441735/7954402/f0dba998-09ec-11e5-9c28-78d88c541416.png)
![screen shot 2015-06-03 at 12 14 28 pm](https://cloud.githubusercontent.com/assets/5441735/7954407/fe6da5ac-09ec-11e5-8f81-da3a4d528517.png)
![screen shot 2015-06-03 at 12 16 20 pm](https://cloud.githubusercontent.com/assets/5441735/7954412/094821f0-09ed-11e5-91c2-f3a57d59c962.png)

**Comment by emotality** [comment]
Where are you importing the header file?

Like you said, you have duplicate files of Reachability, keep my folder and delete the other one.
Add #import "ATAppUpdater.h" no matter the warning.
Clear build with Cmnd+Shift+K
Close Xcode and re-open the project
Build the project with Cmnd+B

Let me know if you still get the errors.. (:

**Comment by jp-dubey07** [comment]
I have followed as per instructions but still getting error "ATAppUpdater.h" file not found. I'm using Xcode 6.3.1.
![screen shot 2015-06-04 at 11 57 40 am](https://cloud.githubusercontent.com/assets/5441735/7978238/b6dde71e-0ab0-11e5-85d2-9bceab864b8e.png)

**Comment by jp-dubey07** [comment]
Now if I use #import "ATAppUpdater/ATAppUpdater.h" then above error is resolved but two new error prompts up shown in below screenshot.
![screen shot 2015-06-04 at 11 59 56 am](https://cloud.githubusercontent.com/assets/5441735/7978274/49be0802-0ab1-11e5-9110-90799c1704e2.png)

**Comment by jp-dubey07** [comment]
Is there any framework to be added? Also, ATAppUpdater.m and Reachability.m file are not added to Compile Sources of Target (screenshot 1). If I try to add it manually it doesn't shows up (screenshot 2).
![screen shot 2015-06-04 at 12 07 06 pm](https://cloud.githubusercontent.com/assets/5441735/7978408/9301c76e-0ab2-11e5-998e-5c11461f376e.png)
![screen shot 2015-06-04 at 12 07 24 pm](https://cloud.githubusercontent.com/assets/5441735/7978409/9358eefe-0ab2-11e5-9be9-45588d56de56.png)

**Comment by emotality** [comment]
That is just strange.. Did you drag and dropped it into the project or did you right click to Add Files then Copy items if needed?

![new_files](https://cloud.githubusercontent.com/assets/2683462/7983434/72c3744e-0ac0-11e5-995e-a348d2cf35ea.png)

Delete all ATAppUpdater files and add it this way, let me know.
So sorry about this! :(

**Comment by emotality** [comment]
I have replicated your issue and got the same issues.

To fix:
Link SystemConfiguration.framework
#import <UIKit/UIKit.h> in ATAppUpdater.h

Let me know if this worked for you as well.

**Comment by jp-dubey07** [comment]
Now, I have deleted existing ATAppUpdater and added using 'Right Click -> Add New File..' Selected 'Copy Items if needed' and 'Create groups' (screenshot 1).

This approach shoes ATAppUpdater.m and Reachability.m file in 'Compile Sources' of Build Settings (screenshot 2).

Now, I have added 'SystemConfiguration.framework' in 'Link Binary with Library' (screenshot 3). Cleaned project and then build application. This prompts 19 issues (screenshot 4). I'm using Xcode 6.3.1 and 
OS X 10.10.3.

![screen shot 2015-06-05 at 12 14 25 pm](https://cloud.githubusercontent.com/assets/5441735/8000880/7a42f9de-0b7d-11e5-95c8-eedfac343451.png)
![screen shot 2015-06-05 at 12 15 02 pm](https://cloud.githubusercontent.com/assets/5441735/8000883/7ed29540-0b7d-11e5-8af3-e9c355a51072.png)
![screen shot 2015-06-05 at 12 15 23 pm](https://cloud.githubusercontent.com/assets/5441735/8000885/7f2856a6-0b7d-11e5-8df5-eb7b0c843c45.png)
